crates/ruff_linter/resources/test/fixtures/pycodestyle/W291.py

```diff
@@ -0,0 +1,2 @@
+'''trailing whitespace 
+inside a multiline string'''
```

Comment:
Do you mind adding a test for a multi-line f-string here?

Reply:
I added a test and it failed! From a brief look, it didn't seem trivial, and I would prefer to do this as a separate PR.

Reply:
Sounds good!